reference/audit-logs.mdx

```diff
@@ -0,0 +1,69 @@
+---
+title: 'Track activity in Axiom'
+description: "This page explains how to track activity in your Axiom organization with audit logs."
+sidebarTitle: Audit Logs
+tags:
+    ['audit logs', 'track activity']
+---
+
+Audit logs allow you to track who did what and when within your Axiom organization.
+
+Tracking activity in your Axiom organization with audit logs is useful for legal compliance reasons. For example, you can investigate the following:
+
+- Track who has accessed the Axiom platform.
+- Track organization access over time.
+- Track data access over time.
+
+Audit logs also make it easier to manage your Axiom organization. They allow you to do the following, among others:
+
+- Track changes made by your team to your observability posture.
+- Track monitoring performance.
+- Track billable usage.
+
+Audit logs are available to all users. Enterprise customers can query audit logs for the full time range. Other customers can query audit logs for the previous three days.
+
+## Explore audit logs
+
+1. Go to the **Explore** tab, and then click **Builder**.
+2. In the list of datasets, click **axiom-audit**.
```

Comment:
Audit log will not appear in a user's list of datasets. They'll need to write an APL query referencing the reserved 'axiom-audit' dataset to access